### PR TITLE
Feature/sdk authentication

### PR DESCRIPTION
**Pull request by easingthemes**
## Description

Add examples for authentication using SDK.

## How Has This Been Tested?
Locally with user:pass Basic auth

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by godanny86** [comment]
@easingthemes since the setupProxy.js is only for local development, does it make sense to include the token example there? Was wondering if the example should be added to `useGraphQL.js`?

**Comment by easingthemes** [comment]
> @easingthemes since the setupProxy.js is only for local development, does it make sense to include the token example there? Was wondering if the example should be added to `useGraphQL.js`?

- `setupProxy.js` is for local development of ReactJS app, but AEM content can be anything: Local, Dev,  Stage, even PROD. Not sure if I understood what should be included there, can you please explain?
-  `useGraphQL.js` - we can't use token here because of security, Client JS should call some servlet which should handle token auth. `setupProxy.js` is our example here,

**Review by godanny86 — APPROVED** [review]

**Comment by godanny86** [comment]
LGTM @easingthemes !

File: react-app/package.json [hunk 1]
```diff
@@ -19,7 +20,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
```
Comment (godanny86) [inline react-app/package.json:23]:
```suggestion
    "eject": "react-scripts eject"
```

**Pull request merged**